modules/accessing-windows-node-using-rdp.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-windows-container-issues.adoc
+
+[id="accessing-windows-node-using-rdp_{context}"]
+= Accessing a Windows node using RDP
+
+You can access a Windows node by using a Remote Desktop Protocol (RDP).
+
+.Prerequisites
+
+* You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
+* You have created a Windows MachineSet.
+* You have added the key used in the `cloud-private-key` secret and the key used when creating the cluster to the ssh-agent. For security reasons, remember to remove the keys from the ssh-agent after use.
+
+.Procedure
+
+. Run the following command to set up an SSH tunnel:
++
+[source,terminal]
+----
+$ ssh -L 2020:<windows_node_internal_ip>:3389 \ <1>
+    core@$(oc get service --all-namespaces -l run=ssh-bastion -o go-template="{{ with (index (index .items 0).status.loadBalancer.ingress 0) }}{{ or .hostname .ip }}{{end}}")
+----
+<1> Specify the internal IP address of the node, which can be discovered by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes <node_name> -o jsonpath={.status.addresses[?\(@.type==\"InternalIP\"\)].address}
+----
+
+. SSH into the Windows node and run the following command to create a password for the user:
++
+[source,terminal]
+----
+C:\> net user <username> * <1>
+----
+<1> Specify the cloud provider username, such as `Administrator` for AWS or `capi` for Azure.
+
+You can now RDP into the Windows node at `localhost:2020` using an RDP client.
```

Comment:
Is  `localhost` going to be `<windows_node_internal_ip>` from within the cluster? How do you connect to it from outside the cluster?

Reply:
Yes it is going to be `localhost`. 
```
$ ssh -L 2020:<windows_node_internal_ip>:3389 \ <1>
    core@$(oc get service --all-namespaces -l run=ssh-bastion -o go-template="{{ with (index (index .items 0).status.loadBalancer.ingress 0) }}{{ or .hostname .ip }}{{end}}")
```
creates a SSH tunnel over which you can RDP into the Windows node.